.stylelintrc

```diff
@@ -0,0 +1,59 @@
+{
+  "rules": {
+    "color-hex-case": "lower",
+    "color-no-invalid-hex": true,
+
+    "function-calc-no-unspaced-operator": true,
+    "function-comma-space-after": "always-single-line",
+    "function-comma-space-before": "never",
+    "function-name-case": "lower",
+    "function-url-quotes": "always",
+    "function-whitespace-after": "always",
```

Comment:
This: a { transform: translate(1, 1) scale(3); } Not: a { transform: translate(1, 1)scale(3); }